src/pages/storage/forms/DuplicateVolumeForm.tsx

```diff
@@ -118,7 +123,7 @@ const DuplicateVolumeForm: FC<Props> = ({ volume, close }) => {
           pool: volume.pool,
           volume_only: !values.copySnapshots,
           // logic from the lxc source code.
-          // We should not set source.project if target project is the same as the source project
+          // We should set source.project or source.location only if the target project/location is NOT the same as the source project
           project:
             values.project !== volume.project ? volume.project : undefined,
           location: volume.location,
```

Comment:
The changed comment seems to be wrong. We set the location regardless of a change below.